lib/ts/axios.ts

```diff
@@ -179,6 +181,9 @@ export function responseInterceptor(axiosInstance: any) {
                 let frontToken = response.headers["front-token"];
                 if (frontToken !== undefined) {
                     await FrontToken.setItem(frontToken);
+                    if (!isNewSession) {
+                        onTokenUpdate();
+                    }
```

Comment:
This makes sense, however there could be another (maybe better) way of doing this: In `FrontToken.setItem`, we check if a token already exists (so a session exists), and then in case the new access token payload is different than what is saved, we fire this event. We would have to be careful about not doing that during session refresh.

What do you think?